CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ﻿# Changelog
 
+## v1.1.0-preview.1
+
+Adds support for durable entities. Learn more [here](https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-entities?tabs=csharp).

```

Comment:
The name of the tab may change to be something specific to out-of-proc, but not a blocker for this PR.

Reply:
Yeah I can update that after we publish docs.